.gitignore

```diff
@@ -24,15 +24,11 @@ __pycache__/
 /ID
 /build*
 /install-tree
-/mkosi.builddir/
-/mkosi.output/
-/mkosi.installdir/
 /mkosi.key
 /mkosi.crt
 /mkosi.tools/
 /mkosi.tools.manifest
```

Comment:
Why they kept here? I'd like to also move them.
Of course, each developer needs to move the files after this PR merged, but I think it is OK.

Reply:
These are mkosi build artifacts, not configuration. If anything, they should be moved into the build directory, but I'd like to make this a separate PR.

Reply:
OK.